spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/hystrix/CustomCommandHook.java

```diff
@@ -0,0 +1,7 @@
+package org.springframework.cloud.netflix.feign.hystrix;
+
+import com.netflix.hystrix.HystrixCommand;
+
+public interface CustomCommandHook {
```

Comment:
I agree, I'll add the javadoc on my next PR which will also inclode the asciidoc update.